src/steps/fetch-app-addons/index.ts

```diff
@@ -67,6 +67,7 @@ export function createAddonEntity(addon, applicationId): Entity {
         applicationId: applicationId,
         category: ['platform'],
         endpoints: [addon.web_url],
+        function: [],
```

Comment:
Updating the SDK required this property. Tbh I'm not really sure whether `heroku_addon` should have been a `Service` in the first place. Lmk if you have suggestions for value(s) for `function`. Nothing stood out from the data-model examples.

Reply:
It looks like add ons are services for the apps folks deploy, https://elements.heroku.com/addons. It would seem the function of each add on would be different, so the code would need to calculate a meaningful value.

I'm on my phone so a bit hard to review other integrations. 😁